### PR TITLE
Add PCL/netstandard documentation

### PR DESCRIPTION
**Pull request by rchande**
Adds documentation for the automatic and manual steps to the PCL portion of this document.

cc @cartermp

### CODE REVIEW

File: docs/core/tutorials/using-on-windows.md [hunk 1]
```diff
@@ -238,7 +238,29 @@ Close the previous solution if it was open: you will be starting a new script fr
...
+
+2.	Remove the dependency on “Microsoft.NETCore”
+
+3.	Modify the dependency on “Microsoft.NETCore.Portable.Compatibility” version “1.0.0” to version “1.0.1”
+
+4.	Add a dependency on “NETStandard.Library” version “1.6.0”
+
+5.	From the “frameworks” element, remove the “dotnet” framework (and the “imports” element within it)
+
+6.	Add ` "netstandard1.x” : { } ` to the frameworks element, where x is replaced with the version of .NET Standard you want to target
```
Comment (Pilchie) [inline docs/core/tutorials/using-on-windows.md]: Is it worth having a single block with a fully formed project.json that will work with PCL here?

**Comment by Pilchie** [comment]
:+1:

File: docs/core/tutorials/using-on-windows.md (hunk 1 above)
Comment (cartermp) [inline docs/core/tutorials/using-on-windows.md]: I'll create an issue.  This whole document could probably be augmented by having full project.json files to supplement the steps.

**Comment by cartermp** [comment]
LGTM.  Thanks, @rchande!

File: docs/core/tutorials/using-on-windows.md (hunk 1 above)
Comment (harikmenon) [inline docs/core/tutorials/using-on-windows.md]: Agreed, Having a sample project.json will help immensely.

**Pull request merged**